salesforce-composite/2.7/modules/ROOT/pages/salesforce-composite-connector-examples.adoc

```diff
@@ -114,21 +115,21 @@ The following is an example of HTTP input for the *Transform message* component:
 
 === Add the Create SObject Tree Operation
 
-Add the *Create sobject tree* operation to connect with Salesforce and create an SObject tree:
+Add the *Create SObject Tree* operation to connect with Salesforce and create a SObject tree:
```

Comment:
We usually match case with the UI, so the original was probably correct (Create sobject tree). You should verify the case in the connector UI (I assume you installed the connector in Studio).

An SObject